README.md

```diff
@@ -1,27 +1,201 @@
+
+There is a default set of URLs that providers may allow your application to use (TODO: link to default URL whitelist).
+In order to use URLs from outside of this whitelist, you need to provide a signed manifest that can be validated by certificates issued by Golem.
+
+**NOTE:** Currently there is no process for obtaining the certificate needed to validate the manifest signature. Please contact us on discord if you need to use URLs outside the default whitelist.
+
+You can use this command multiple times to update URLs in the manifest, and you can pass multiple URLs at once.
+
+
+#### Example: Simple use
+
+This command will update the manifest file with the URL.
+
+```shell
+golem-sdk manifest net add-outbound https://golem.network
+```
+
+
+#### Example: Multiple URLs
+
+This command will update the manifest file with all the URLs provided.
+
+```shell
+golem-sdk manifest net add-outbound https://golem.network https://github.com https://example.com
+```
+
+### Signing the manifest
+
+In order to use URLs outside the default whitelist, you need to sign the manifest with a key provided by Golem.
+
+**NOTE:** Currently there is no process for obtaining the certificate needed to validate the manifest signature. Please contact us on discord if you need to use URLs outside the default whitelist.
+
+If your private key is encrypted, you will need to provide the correct passphrase (`-p` or `--passphrase` option).
+
+To sign the manifest, run:
+```shell
+golem-sdk manifest sign -k <private-key>
+```
+
+This command will produce a signature file (by default `manifest.sig`) that you will need to use in your application.
+
+
+### Verifying the signature
+
+You can verify manifest signature with your certificate using the following command:
+
+```shell 
+golem-sdk manifest verify
+```
+
+By default, it will use `manifest.pem` as the certificate file and `manifest.sig` as the signature file. You can change that by using `--certificate-file` and `--signature-file` options.
+
+On success, it will print the following message:
+
+```
+Manifest matches signature.
+```
+
+It is important to use this command to make sure the key you are using is compatible with your certificate.
+
+
+## Contributing
+
+If you'd like to contribute to the project, you can fork the repository and create a Pull Request.
+Code contribution are warmly welcomed.
+
+Please make sure the code follows coding style as configured in `.eslintrc` and `.prettierrc`.
+
+The Pull Request should describe what changes you've made, what's the purpose of them and how to use them.
+
+
+## Links
+
+- Project homepage: https://github.com/golemfactory/golem-sdk-cli
+- Repository: https://github.com/golemfactory/golem-sdk-cli
+- Issue tracker: https://github.com/golemfactory/golem-sdk-cli/issues
+    - In case of sensitive bugs like security vulnerabilities, please contact
+      my@email.com directly instead of using issue tracker. We value your effort
```

Comment:
myemail needs to be replaced. Did you figure out anything with Staziek?

Reply:
he didn't reply